src/main/java/com/paneedah/weaponlib/PlayerTransitionProvider.java

```diff
@@ -3,59 +3,22 @@
 
-    public static class Builder {
-
-        private List<MultipartTransition<Part, RenderContext<RenderableState>>> proningTransitions = new ArrayList<>();
-        
-        private List<MultipartTransition<Part, RenderContext<RenderableState>>> proningAimingTransitions = new ArrayList<>();
-
-        public Builder withProningTransition(MultipartTransition<Part, RenderContext<RenderableState>> transition) {
-            proningTransitions.add(transition);
-            return this;
-        }
-        
-        public Builder withProningAimingTransition(MultipartTransition<Part, RenderContext<RenderableState>> transition) {
-            proningAimingTransitions.add(transition);
-            return this;
-        }
-
-        public PlayerTransitionProvider build() {
-            return new PlayerTransitionProvider(this);
-        }
-    }
-    
-    private List<MultipartTransition<Part, RenderContext<RenderableState>>> normalTransitions = 
-            Arrays.asList(
-                    new MultipartTransition<Part, RenderContext<RenderableState>>(
-                            Part.MAIN, renderContext -> {}, 200, 0)
-                    .withPartPositionFunction(Part.LEFT_HAND, rc -> {})
-                    .withPartPositionFunction(Part.RIGHT_HAND, rc -> {})
-                    .withPartPositionFunction(Part.LEFT_LEG, rc -> {})
-                    .withPartPositionFunction(Part.RIGHT_LEG, rc -> {})
-                    );
-
-    private Builder builder;
-
-    private PlayerTransitionProvider(Builder builder) {
-        this.builder = builder;
-    }
-    
-    protected PlayerTransitionProvider() {}
-
     @Override
     public List<MultipartTransition<Part, RenderContext<RenderableState>>> getTransitions(RenderableState state) {
-        if(state == RenderableState.PRONING) {
-            return builder.proningTransitions;
-        } else if(state == RenderableState.PRONING_AIMING) {
-            return builder.proningAimingTransitions;
-        } else {
-            return normalTransitions;
-        }
+        // Panda: This absolutely sucks. Why are we doing this to begin with? There's certainly better ways. It took me absolutely forever to
+        //        figure out this spaghetti mess of fucking MultipartTransition lists and MultipartTransitionProviders. Fuck me sideways
+        //        this is absolute cancer and there is about a 0% chance this is efficient. But oh well - whatever. Gets the job done for now.
+        // TODO: CLEAN THIS UP.
+        return Collections.singletonList(
+                new MultipartTransition<Part, RenderContext<RenderableState>>(Part.MAIN, renderContext -> {}, 200, 0)
+                        .withPartPositionFunction(Part.LEFT_HAND, rc -> {})
+                        .withPartPositionFunction(Part.RIGHT_HAND, rc -> {})
+                        .withPartPositionFunction(Part.LEFT_LEG, rc -> {})
+                        .withPartPositionFunction(Part.RIGHT_LEG, rc -> {})
+        );
```

Comment:
What the hell????
Keep:
```java
return normalTransitions;
```
Without using that constant you would be recreating the transition every time `getTransitions` is called!